manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -169,6 +169,10 @@ type M4DApplicationStatus struct {
 	// Generated resource identifier
 	// +optional
 	Generated *ResourceReference `json:"generated,omitempty"`
+
+	// Provisioned storage (Dataset resources)
+	// +optional
```

Comment:
A better description of what this is would be helpful.  When is it used? It's mapping what to what?

Reply:
Added.